database/migrations/20220618150230-create-new.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('News', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      id: {
+        type: Sequelize.INTEGER
```

Comment:
Gabi mencionó que es buena práctica tener explícito lo de NOT NULL, entonces yo se lo añadiría